enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -577,6 +579,11 @@ func (r *Env) addExecutor(options *ExecutorOptions) *Executor {
 	env.SetRemoteExecutionClient(repb.NewExecutionClient(clientConn))
 	env.SetSchedulerClient(scpb.NewSchedulerClient(clientConn))
 	env.SetAuthenticator(r.newTestAuthenticator())
+	xl, err := xcode.NewXcodeLocator()
+	if err != nil {
+		assert.FailNowf(r.t, "Failed to set XCodeLocator: %s", err.Error())
+	}
+	env.SetXcodeLocator(xl)
```

Comment:
will this cause issue when run on linux?

Reply:
It shouldn't. This is the same code as in enterprise/server/cmd/executor/executor.go. If it's on linux, code in xcode.go will be invoked, and if it's on mac, xcode_darwin.go will be invoked. 

The remote_execution_test succeeds in the workflow; so I assume it works on linux.